crates/re_viewer/src/ui/view_spatial/scene/scene_part/meshes.rs

```diff
@@ -37,19 +37,19 @@ impl MeshPart {
 
         let _default_color = DefaultColor::EntityPath(ent_path);
         let world_from_obj_affine = glam::Affine3A::from_mat4(world_from_obj);
-        let entity_outlines = highlights.entity_outline_mask(ent_path.hash());
+        let entity_highlight = highlights.entity_outline_mask(ent_path.hash());
```

Comment:
The intermixing of "highlight" and "outline" as concepts could probably use some attention at some point.

Reply:
as you can see I went back and forth a bit: They are a _form of_ highlights and soon the only one that's why I went back to calling it highlight 🤔